src/Orchard.Web/Modules/Orchard.Email/Views/EditorTemplates/Parts/SmtpSettings.cshtml

```diff
@@ -25,10 +26,29 @@
         <span class="hint">@T("The SMTP server port, usually 25.")</span>
     </div>
     <div>
-        @Html.EditorFor(m => m.EnableSsl)
-        <label for="@Html.FieldIdFor(m => m.EnableSsl)" class="forcheckbox">@T("Enable SSL communications")</label>
-        @Html.ValidationMessage("EnableSsl", "*")
-        <span class="hint">@T("Check if the SMTP server requires SSL communications.")</span>
+        <label for="@Html.FieldIdFor(m => m.EncryptionMethod)">@T("Encryption method to use")</label>
+        <select for="@Html.FieldIdFor(m => m.EncryptionMethod)" name="@Html.NameOf(m => m.EncryptionMethod)" disabled="@Model.AutoSelectEncryption">
+            @Html.SelectOption(
+                Model.EncryptionMethod,
+                SmtpEncryptionMethod.None,
+                String.Join(String.Empty, T("None").ToString(), " - ", T("Connect to server using insecure connection.").ToString()))
+            @Html.SelectOption(
+                Model.EncryptionMethod,
+                SmtpEncryptionMethod.None,
```

Comment:
This may be a error, SmtpEncryptionMethod.None is the value of all three select options.

Reply:
@dministro and the correct value isn't persisted, even if I correct the technical values here, please check.

Reply:
@dministro just commenting a reminder here for your SendGrid testing.

Reply:
Sorry, I just noticed that it's fixed already!